internal/boxcli/root.go

```diff
@@ -83,6 +84,10 @@ func Main() {
 		code := sshshim.Execute(os.Args)
 		os.Exit(code)
 	}
+	if len(os.Args) > 1 && os.Args[1] == "bug" {
+		telemetry.ReportErrors()
+		return
+	}
```

Comment:
Is this just for performance? Or to avoid having to deal with a cycle? 

Can you add comment.